src/Jackett.Common/Indexers/Libble.cs

```diff
@@ -0,0 +1,321 @@
+                   link: "https://libble.me/",
+                   caps: new TorznabCapabilities
+                   {
+                       TVSearchAvailable = false,
+                       SupportedMusicSearchParamsList = new List<string> { "q", "album", "artist", "label", "year" }
+                   },
+                   configService: configService,
+                   client: wc,
+                   logger: l,
+                   p: ps,
+                   configData: new ConfigurationDataBasicLogin())
+        {
+            Encoding = Encoding.UTF8;
+            Language = "en-us";
+            Type = "private";
+
+            AddCategoryMapping(1, TorznabCatType.Audio, "Music");
+            AddCategoryMapping(2, TorznabCatType.Audio, "Libble Mixtapes");
+            AddCategoryMapping(7, TorznabCatType.AudioVideo, "Music Videos");
+        }
+
+        public override async Task<ConfigurationData> GetConfigurationForSetup()
+        {
+            await RequestStringWithCookies(LandingUrl);
+            return configData;
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+
+            var pairs = new Dictionary<string, string>
+                        {
+                            {"username", configData.Username.Value},
+                            {"password", configData.Password.Value},
+                            {"login", "Login"}
+                        };
+
+            var result = await RequestLoginAndFollowRedirect(LoginUrl, pairs, null, true, SearchUrl, LandingUrl, true);
+            await ConfigureIfOK(result.Cookies, result.Content?.Contains("logout.php") == true,
+                () =>
+                {
+                    var parser = new HtmlParser();
+                    var dom = parser.ParseDocument(result.Content);
+                    var warningNode = dom.QuerySelector("#loginform > .warning");
+                    var errorMessage = warningNode?.TextContent.Trim().Replace("\n\t", " ");
+                    throw new ExceptionWithConfigData(errorMessage, configData);
+                });
+
+
+            SaveConfig();
+
+            return IndexerConfigurationStatus.RequiresTesting;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            // https://libble.me/torrents.php?searchstr=the+used&taglist=&tags_type=1&order_by=time&order_way=desc
+            var releases = new List<ReleaseInfo>();
+            var searchString = query.GetQueryString();
+            var prevCook = CookieHeader + "";
+            var searchUrl = SearchUrl;
+
+            var searchParams = new Dictionary<string, string> { };
+            var queryCollection = new NameValueCollection { };
+
+            // Skip search params if its a test, ask for all torrents
+            if (!query.IsTest)
```

Comment:
I think you don't need this check. The code inside this condition will work well in regular and test queries.